src/foundation/layout.css

```diff
@@ -1,5 +1,15 @@
 :root {
-  --page-padding-x: 6rem;
+  --main-content-max-width: 100rem;
+  --page-padding-x: calc((100vw - var(--main-content-max-width)) / 2);
   --page-padding-y: 3rem;
-  --main-content-percentage-width: calc(2 / 3);
+  --main-content-ratio: calc(2 / 3);
+  --main-content-aside-gap: 1rem;
```

Comment:
I would use 3rem but I am sure what this is.

Reply:
You mean you are not sure?

It's column number 2

![image](https://user-images.githubusercontent.com/12775440/102086646-b2830400-3e18-11eb-96fd-8e81fcc1d6c3.png)

Reply:
If it's `grid-column-gap` let's double it.